docs/user-guides/user-guide-gui.md

```diff
@@ -0,0 +1,83 @@
+# Use the Web Interface to Find and Pull Images
+
+> :point_right: Using a browser, you can browse a **zot** repository and download container images and artifacts.
```

Comment:
Maybe another way of expressing this is necessary, the GUI doesn't directly allow the user to download container images, rather, it provides the the user with the necessary CLI commands to do so

Reply:
Suggest:

"Use the Web Interface to Find Images

Using a browser, you can browse a zot repository for container images and artifacts. The web interface also provides the commands for downloading an image using popular third-party tools."